docs/csharp/quick-starts/branches-and-loops.yml

```diff
@@ -268,7 +269,9 @@ items:
     > This online coding experience is in preview mode. If you encounter problems, please report them [on the dotnet/try repo](https://github.com/dotnet/try/issues).
 
   content: |
-    You've completed the "branches and loops" tutorial. You can learn more about these concepts in these topics:
+    You've completed the "branches and loops" quick start. You can learn more about these concepts in these topics:
+
+    You can continue these quick starts on your own development environment. The .NET topic [Get Started in 10 minutes](https://www.microsoft.com/net/core) has instructions for setting up your local development environment. Once you've done that, continue with [List collection quickstart](arrays-and-collections.md) in your local environment.
```

Comment:
This paragraph interrupts the previous paragraph and the list of topics that's supposed to follow it.